test/Microsoft.IdentityModel.Protocols.Tests/ExtensibilityTests.cs

```diff
@@ -92,12 +92,12 @@ public async Task ConfigurationManagerUsingCustomClass()
             // Wait for the refresh to complete.
             await Task.Delay(250).ContinueWith(_ =>
             {
-
                 configuration2 = configManager.GetConfigurationAsync().GetAwaiter().GetResult();
-                if (IdentityComparer.AreEqual(configuration.Issuer, configuration2.Issuer))
-                    context.Diffs.Add("IdentityComparer.AreEqual(configuration.Issuer, configuration2.Issuer)");
             });
 
+            if (IdentityComparer.AreEqual(configuration.Issuer, configuration2.Issuer))
```

Comment:
I don't understand, how do you know the refresh completed?

Reply:
On my local I just printed when configuration refresh started and completed to check what was happening on the code but, couldn't get it to fail like it is on the build so, this would wait until the entire continuation task has completed before doing the check.